src/engine/profiler.cc

```diff
@@ -180,8 +184,15 @@ void Profiler::DumpProfile() {
 
 
 inline uint64_t NowInUsec() {
+#if defined(_MSC_VER) && _MSC_VER <= 1800
+  LARGE_INTEGER frequency, counter;
+  QueryPerformanceFrequency(&frequency);
+  QueryPerformanceCounter(&counter);
+  return counter.QuadPart * 1000000 / frequency.QuadPart;
```

Comment:
I'm not familiar with Windows API. Just to confirm, what's the time unit here? We should stay the same with chrono::microseconds